src/tests/080-commentTest.js

```diff
@@ -141,4 +141,54 @@ describe('User/manager should be able to post/get comments', () => {
         done(err);
       });
   });
+  it('It should not delete comment if token is invalid', (done) => {
+    chai.request(app)
+      .delete('/api/v1/trips/delete/1')
+      .set('token', invalidToken)
+      .send(mockData.userComment)
+      .end((err, res) => {
+        console.log('this is my invlaid token', res.body);
+        expect(res.status).eql(401);
+        expect(res.body.message.message).eql('jwt malformed');
+        done(err);
+      });
+  });
+
+  it('It should not delete comment if comment id is not an integer', (done) => {
+    chai.request(app)
+      .delete('/api/v1/trips/delete/dffghj')
+      .set('token', userToken)
+      .send(mockData.userComment)
+      .end((err, res) => {
+        expect(res.status).eql(400);
+        expect(res.body.message[0]).eql('Comment Id should be an integer');
+        done(err);
+      });
+  });
+
+  it('It should not delete comment if does not own the comment', (done) => {
+    chai.request(app)
+      .delete('/api/v1/trips/delete/1')
+      .set('token', managerToken)
+      .send(mockData.userComment)
+      .end((err, res) => {
+        console.log(res.body);
+
+        expect(res.status).eql(200);
+        expect(res.body.message).eql('Your comment has been deleted successfully');
+        done(err);
+      });
```

Comment:
I think the `It()` method has an incorrect message.  The deletion is successfully but the message does not match `It should not delete comment if does not own the comment`.